.github/workflows/build-docker-image.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
       - name: Set build number
         id: ntbs_build_step
-        run: echo '::set-output name=NTBS_BUILD::${{ format('build-{0}-{1}', github.run_number, github.sha) }}'
+        run: echo '::set-output name=NTBS_BUILD::${{ format('feature-branch-build-{0}-{1}', github.run_number, github.sha) }}'
```

Comment:
The `github.run_number` is specific to the workflow that you're running, so will clash with the main test and deploy workflow. Obviously the hashes won't clash, but to avoid confusion I thought it best to change the prefix here.

I considered having the branch name as the prefix, but my feeling is that it would add more difficulty than value, because you lose the fact that alphabetical order is the same as chronological.